src/main/java/edu/internet2/middleware/grouper/changeLog/consumer/Office365ApiClient.java

```diff
@@ -218,6 +239,7 @@ public void addGroup(Group group) {
             try {
                 logger.debug("**** ");
                 String groupName = getGroupName(group);
+                logger.error("groupName = " + groupName);
```

Comment:
this looks like logging used for debugging

Reply:
I removed the line.